components/logo/MailLogo.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { PLAN_NAMES } from 'proton-shared/lib/constants';
+
+const planNames = ['', ...Object.keys(PLAN_NAMES)];
```

Comment:
I don't think you need to add `''` since `planName` prop is not required.

Reply:
mmhh, that's a good point!